services/brig/deb/opt/brig/templates/de/provider/email/activation.txt

```diff
@@ -1,29 +1,31 @@
+[${brand_logo}]
+
+${brand_label_url} [${brand_url}]
+
 BESTÄTIGE DEINE E-MAIL-ADRESSE
-Deine E-Mail-Adresse ${email} wurde benutzt, um sich als Wire Serviceanbieter zu
-registrieren.
+Deine E-Mail-Adresse ${email} wurde benutzt, um sich als ${brand_service}
+Serviceanbieter zu registrieren.
 
 Um die Registrierung abzuschließen, bestätige bitte deine E-Mail-Adresse, indem
 du auf den unteren Button klickst.
 
-Bitte beachte, dass das Service-Provider-Konto nach der Bestätigung der
-E-Mail-Adresse noch durch uns freigeschaltet werden muss. Dies erfolgt
-üblicherweise innerhalb von 24 Stunden. Über die Freischaltung wirst du per
-E-Mail informiert.
+Bitte beachte, dass das ${brand_service} nach der Bestätigung der E-Mail-Adresse
+noch durch uns freigeschaltet werden muss. Dies erfolgt üblicherweise innerhalb
+von 24 Stunden. Über die Freischaltung wirst du per E-Mail informiert.
 
 Überprüfe [${url}]Falls du nicht auf den Button klicken kannst, kopiere den Link
 und füge ihn in deinem Browser ein:
 
 ${url}
 
-Solltest du dich nicht für eine Wire Service-Provider-Konto mit dieser
-E-Mail-Adresse registriert haben, kannst du diese Meldung ignorieren. Falls du
-einen Missbrauch deiner E-Mail-Adresse melden möchtest kontaktiere uns bitte.
-[https://support.wire.com/hc/de/requests/new]
+Solltest du dich nicht für eine ${brand} Service-Provider-Konto mit dieser
```

Comment:
Seems weird that it doesn't match activation-subject.txt (`${brand_service}-Benutzerkonto`).